climpred/bootstrap.py

```diff
@@ -381,9 +444,11 @@ def bootstrap_hindcast(
     reference,
     metric='pearson_r',
     comparison='e2r',
+    dim='init',
     sig=95,
     bootstrap=500,
     pers_sig=None,
+    **metric_kwargs,
 ):
     """Wrapper for bootstrap_compute for hindcasts."""
```

Comment:
Can you copy and paste the docstring from `bootstrap_compute` to here? Currently this isn't friendly at all to new users. Here is what they see when trying to figure out how to use the function: https://climpred.readthedocs.io/en/latest/api/climpred.bootstrap.bootstrap_hindcast.html#climpred.bootstrap.bootstrap_hindcast

Reply:
I think it would be good also in the docstring to have a bullet list for `metric_kwargs` to show exactly what keywords are expected for which metrics. Otherwise they have to scroll through the raw code to try to figure that out.

Reply:
towards 1): ok. but now changes in docs will be needed to be done three times.
towards 2): no, this will be duplication. I will refer users to reading the metrics.rst.